spec/ability/roles_spec.rb

```diff
@@ -44,6 +44,14 @@
     FactoryGirl.create(:flagged_scanned_resource, user: image_editor, identifier: ['ark:/99999/fk4445wg45'])
   }
 
+  let(:complete_ephemera_folder) {
```

Comment:
I don't see any assertions about this object — do we need it?  Or do you want to add some assertions about being able to access a complete object?